src/main/java/xyz/rc24/bot/commands/wii/ErrorInfoCmd.java

```diff
@@ -257,7 +257,6 @@ private void success(@NotNull CommandEvent event, @NotNull Response response)
     private final Map<Integer, String> codeNotes = new HashMap<Integer, String>()
     {{
         put(101409, "If you are getting this problem while doing something with Wii Mail, check if you patched the nwc24msg.cfg correctly. https://bit.ly/2QUrsyD");
-        put(102032, "The IOS the app/game uses is not patched for RiiConnect24. Try sending a message again but do it quickly, you need to do it in less than a minute. We hope to improve Wii Mail to stop getting this error.");
```

Comment:
Not sure.... I doubt that error will never seen again

Reply:
well, I guess we can remove the last part about improving wii mail at the very least

Reply:
Ok, remove the "Try sending a message...." part